core_backend/app/question_answer/schemas.py

```diff
@@ -13,9 +14,9 @@ class QueryBase(BaseModel):
     """
 
     query_text: str = Field(..., examples=["What is AAQ?"])
-    session_id: Optional[int] = None
     generate_llm_response: bool = Field(False)
     query_metadata: dict = Field({}, examples=[{"some_key": "some_value"}])
+    session_id: SkipJsonSchema[int | None] = Field(default=None, exclude=True)
```

Comment:
We can do this to remove optional fields from Swagger UI (but MUST BE OPTIONAL!)

Reply:
Nice!